# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/spellchecker/TextServicesDocument.h
# Commit: ade9a3381ccb
# Full Hash: ade9a3381ccb219d2ae1b66c2b2c162c287cafce
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2021-07-13 21:42:32
# Regressor Bug: 1718924
# File Overlap Count: 2
# Description:
#   Bug 1718924 - part 1: Get rid of `TextServicesDocument::IsTextNode()` and use `AsText()` if it's safe r=m_kato
#   
#   It seems that it treats mainly a text node in various places, but it's not
#   guaranteed by any variable declarations.  So, first of all, I'd like to make
#   it clearer.
# ==============================================================================

diff -r 28c7b58df8e4 -r ade9a3381ccb editor/spellchecker/TextServicesDocument.h
--- a/editor/spellchecker/TextServicesDocument.h	Tue Jul 13 06:01:37 2021 +0000
+++ b/editor/spellchecker/TextServicesDocument.h	Tue Jul 13 08:13:11 2021 +0000
@@ -271,7 +271,6 @@
   nsresult GetFirstTextNodeInNextBlock(nsIContent** aContent);
 
   static bool IsBlockNode(nsIContent* aContent);
-  static bool IsTextNode(nsIContent* aContent);
 
   static bool DidSkip(FilteredContentIterator* aFilteredIter);
   static void ClearDidSkip(FilteredContentIterator* aFilteredIter);
